bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -175,7 +175,7 @@
 	</channel-type>
 
 	<channel-type id="signalQuality">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unit="%">Number</item-type>
```

Comment:
Shouldn't all the README files be updated to match?
![image](https://user-images.githubusercontent.com/26532542/219255781-19317b8a-e490-4e68-adac-176bdfa08170.png)